index.js

```diff
@@ -25,7 +25,7 @@ mount('body', app.start())
 
 function mainView () {
   return html`
-    <body class="App pa4">
+    <body class="pa4">
```

Comment:
We no longer need the `App` class because `choo` mounts directly onto `body` and knows how to handle that (: - accidentally left this in hah